src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -380,6 +383,16 @@ public Builder runDevAppServerWait(int runDevAppServerWaitSeconds) {
     }
 
     /**
+     * Causes the generated gcloud subprocess to inherit the calling process's stdout and stderr.
+     *
```

Comment:
There should probably be a note here saying that output listeners are not supported if this is true.